File: src/main/java/kr/co/lunatalk/domain/exhibition/repository/ExhibitionRepositoryCustom.java
package kr.co.lunatalk.domain.exhibition.repository;

import kr.co.lunatalk.domain.exhibition.domain.Exhibition;
import kr.co.lunatalk.domain.exhibition.domain.ExhibitionVisibility;

import java.time.LocalDateTime;
import java.util.List;

public interface ExhibitionRepositoryCustom {
	List<Exhibition> findActiveExhibitions(ExhibitionVisibility visibility, LocalDateTime now);
}

File: src/main/kotlin/kr/co/lunatalk/domain/exhibition/repository/ExhibitionRepositoryCustom.kt
package kr.co.lunatalk.domain.exhibition.repository

import kr.co.lunatalk.domain.exhibition.domain.Exhibition
import kr.co.lunatalk.domain.exhibition.domain.ExhibitionVisibility
import java.time.LocalDateTime

interface ExhibitionRepositoryCustom {
    fun findActiveExhibitions(visibility: ExhibitionVisibility, now: LocalDateTime): List<Exhibition>
}
